### PR TITLE
Create initial Open API swagger YAML. Closes #40.

### LINKED ISSUE #40: Create swagger.yml file to document our API (use OpenAPI v3)
As a developer, I want to be able to share our API via Swagger so that it can be consumed by other customer or they can generate client code using the defined API.

We plan on sharing/composing APIs across services using the Open API definition

## Acceptance Criteria

- [ ] Verify that a basic swagger.yml file exist describing the current API

### PR DESCRIPTION
**Pull request by chambridge**
* Document status endpoint.

### CODE REVIEW

File: docs/source/specs/openapi.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+openapi: "3.0.0"
```
Comment (adberglund) [inline docs/source/specs/openapi.yml]: I'm pretty swagger/openAPI ignorant. What's generating this YAML?

Comment (chambridge) [reply docs/source/specs/openapi.yml]: Nothing is generating this YAML.

I'm actually in favor of this being a part of "documentation-driven development". You can have an issue to define the API contract for the UI to use and they are then able to use a mock of that API while the implementation is developed. QE can also use this doc to write test cases while development is in progress (ideally).

Comment (adberglund) [reply docs/source/specs/openapi.yml]: Yep, totally into it. Creating stubs to unblock is 👍 . 

The root of my question was around that contract. If the swagger YAML always comes first, then there's way less risk of forgetting to update it with API changes.

**Comment by chambridge** [comment]
Verified RTD build fix: 
http://saltcellar.readthedocs.io/en/40-swagger-api/
Will remove version after this is merged.

**Pull request merged**